### DISCUSSION TITLE
why aren't symlinks rendered?

### DISCUSSION THREAD

**Discussion by vitaly** [nvim-neo-tree/neo-tree.nvim#541] [General]
I see that the symlink_target was commented out in the container component...
1. why? ;)
2. how to switch it back without completely specifying the whole container config in my setup?

**Comment by cseickel** [comment]
> 1. why? ;)

It's just not something that everyone wants to see. There are enough people that would be interested to leave the commented out lines there to let you know it exists, but not enough people to turn it on by default.

> 2\. how to switch it back without completely specifying the whole container config in my setup?

Just copy and paste that one component into your renderer. For example:

```lua
  renderers = {
    directory = {
      { "indent" },
      { "icon" },
      { "current_filter" },
      { "name" },
      { "symlink_target", highlight = "NeoTreeSymbolicLinkTarget" },
      { "clipboard" },
      { "diagnostics", errors_only = true },
      { "git_status" },
    },
    file = {
      { "indent" },
      { "icon" },
      { "name" },
      { "symlink_target", highlight = "NeoTreeSymbolicLinkTarget" },
      { "clipboard" },
      { "bufnr" },
      { "modified" },
      { "diagnostics" },
      { "git_status" },
    },
```
